docs/sphinx/reference-frontend-api.rst

```diff
@@ -365,6 +395,35 @@ Functions
 
 ---------------------------------------
 
+.. function:: void *obs_frontend_add_tools_menu_qaction(const char *name)
+
+   Adds a QAction to the tools menu then returns it.
+
+   :param name: The name for the new menu item.
+   :return: A pointer to the added QAction.
+
+---------------------------------------
+
+.. function:: void obs_frontend_add_tools_menu_item(const char *name, obs_frontend_cb callback, void *private_data)
+
+   Adds a tools menu item and links the ::clicked signal to the
+   callback.
+
+   :param name: The name for the new menu item.
+   :param callback: Callback to use when the menu item is clicked
+   :param private_data: Private data associated with the callback.
+
+---------------------------------------
+
+.. function:: void* obs_frontend_add_dock(void *dock)
+
+   Adds a QDockWidget to the UI (Docks menu).
```

Comment:
Why is "Docks menu" in parentheses here?  Where is the dock actually being added?  Is it just adding an entry to the Docks menu?  Or does this actually create a QDockWidget object and show it?